### ISSUE TITLE
[Bug] Appium tests (iOS). UI elements are missing from page source XML representation with Xamarin 4.6 update

### ISSUE THREAD

**Issue by ljoshenjka** [xamarin/Xamarin.Forms#11026]
### Description
I am using Appium for test automation on Android/iOS apps build with Xamarin.Forms. Appium returns a XML representation of a page UI elements with some properties and gives ability to interact with it.
After updating Xamarin from 4.5 to 4.6 (4.6.0.800) some of the iOS (13) application pages returned by Appium are almost with all UI elements missing from XML page source representation. No issues observed on Android. 

### Steps to Reproduce
1. Open application and inspect it with Appium GUI tool

<img width="884" alt="Screenshot 2020-05-28 at 16 30 54" src="https://user-images.githubusercontent.com/31776300/84486141-a3729c00-aca5-11ea-909c-3665194ab297.png">

### Expected Behavior
All the UI elements are returned in page source XML representation

### Actual Behavior
Page source missing most of the page UI elements

### Basic Information

- Version with issue: Xamarin.Forms 4.6.0.800
- Affected Devices: all iOS
- Appium version: 1.17
- Xcode version: 11.5
- Desktop OS/version used to run Appium: macOS Catalina 10.15.4

**Comment by hartez** [comment]
@ljoshenjka Your issue's Basic Information says all iOS devices are affected, but your description mentions iOS 13 specifically. Are you seeing this on other iOS versions?

**Comment by ljoshenjka** [comment]
@hartez issue is not related to some specific iOS version. I guess I just wanted to point that it could be reproduced even with latest iOS 13.5.1

**Comment by samhouts** [comment]
see also #6667

**Comment by PureWeen** [comment]
@ljoshenjka do you have a repro? or can you post your XAML here?

**Comment by ljoshenjka** [comment]
@PureWeen here is the xaml for the page from screenshot: https://gist.github.com/ljoshenjka/29bb667118d4322b8d436ccdd6fe0d60

**Comment by adamhill** [comment]
I would like to leave a piece of anecdotal information about this bug.

Us on the team at Paycor had problems like this with Xamarin and Appium quite frequently, mainly with finding children / data in Listviews. It was universally fixed with setting ```AutomationProperties.IsInAccessibleTree``` to ```false``` & quite repeatable.

Terrible for A11y but leadership did not have that as a priority.

**Comment by PureWeen** [comment]
The nuget for this Issue is here
https://dev.azure.com/xamarin/public/_build/results?buildId=21211&view=artifacts&type=publishedArtifacts

If anyone wants to test and let me know if that fixes for you

**Comment by alallo** [comment]
@PureWeen that fixed the issue for us. Thanks

**Comment by srikcgaa2** [comment]
This unofficial nuget fixes many of our issues as well on iOS, including the [Issue 11392](https://github.com/xamarin/Xamarin.Forms/issues/11392). Any update which version of Xamarin.Forms this will be available (4.7 or 4.8) and tentatively when we can expect this in an official Xamarin.Forms release?

**Comment by alallo** [comment]
It's already available in the latest 4.8 pre release version if you want to test it
